src/reducers/sources.js

```diff
@@ -445,7 +446,9 @@ export function getHasSiblingOfSameName(state: OuterState, source: ?Source) {
     return false;
   }
 
-  return getSourcesUrlsInSources(state, source.url).length > 1;
+  const urls = getSourcesUrlsInSources(state, source.url);
+  const { origin, pathname } = parseURL(source.url);
+  return urls.filter(url => url.includes(`${origin}${pathname}`)).length > 1;
```

Comment:
It seems like @darkwing has a different solution for this same issue. Will update mine once https://github.com/devtools-html/debugger.html/pull/7303 lands